GiEnJul/Controllers/AdminController.cs

```diff
@@ -138,5 +138,17 @@ public async Task<ActionResult> DeleteGiverAsync([FromBody] DeleteGiverDto giver
             var deletedGiver = await _giverRepository.DeleteAsync(giverToDelete);
             return Ok();
         }
+
+        [HttpGet("Suggestions/Giver/{quantity}")]
+        [HttpGet("Suggestions/Giver")]
+        public async Task<IList<GiverDataTableDto>> GetSuggestedGiversAsync([FromBody] string location, int quantity = 1)
+        {
+            if (quantity < 1) throw new ArgumentOutOfRangeException();
+
+            var activeEvent = await _eventRepository.GetActiveEventForLocationAsync(location);
+            var suggestions = await _giverRepository.GetSuggestionsAsync(activeEvent, location, quantity);
+
```

Comment:
```suggestion
var givers = await _giverRepository.GetSuggestionsAsync(activeEvent, location, quantity);
var suggestions = SuggestionHelper.GetSuggestions(givers, quantity);
```

Alternative is to call Helper-function here. To keep the repository to SOLID principles.